develop/tutorials/articles/115-mobile/01-android-apps-with-liferay-screens/07-creating-android-screenlets.markdown

```diff
@@ -177,29 +179,29 @@ to handle user actions. Follow these steps to create this View class:
     has a reference to the Screenlet. For example, here are `AddBookmarkView`'s 
     instance variables: 
 
-        private EditText _urlText;
-        private EditText _titleText;
-        private BaseScreenlet _screenlet;
+        private EditText urlText;
```

Comment:
Yup! They did away with that practice in Screens 2.0, as well as putting instance variables at the bottom of the class.